Werewolf for Telegram/Shared/GameBalancing.cs

```diff
@@ -207,6 +207,8 @@ public static int GetStrength(this IRole role, List<IRole> allRoles)
             {
                 case IRole.Villager:
                     return 1;
+                case IRole.ScapeGoat:
+                    return 1;
```

Comment:
Before, the new roles were always added at the end of the list, so you should stick to that.
The strength of 1 is arguable but since the Wolf Man has 1 too (why? 🤔 ) it's probably acceptable.